lib/rucio/core/replica.py

```diff
@@ -1111,10 +1109,10 @@ def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns,
 
                         if 'site' in client_location and client_location['site']:
                             # is the RSE site-configured?
-                            rse_site_attr = get_rse_attribute('site', rse_id, session=session)
+                            rse_site_attr = get_rse_attribute(rse_id, 'site', session=session)
                             replica_site = ['']
```

Comment:
This variable is defined as a list here, but reassigned as a string just below. Probably should be initialized to `None`?